config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -130,6 +130,47 @@ periodics:
       secret:
         secretName: k8s-triage-robot-github-token
 
+- name: ci-k8s-triage-robot-closed-triage
+  interval: 10m
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  annotations:
+    testgrid-dashboards: sig-contribex-k8s-triage-robot
+    description: A job to remove needs-triage from closed issues/PRs that are marked as requiring triage.
+    testgrid-tab-name: triage
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20220907-aef2c49444
+      command:
+      - commenter
+      args:
+      - |-
+        --query=org:kubernetes
+        org:kubernetes-sigs
+        org:kubernetes-client
+        org:kubernetes-csi
+        is:closed
+        is:unmerged
```

Comment:
Another way to achieve this PR maybe could be to "react" to a closing webhook and then do the label change there itself instead of doing a sweep periodically but probably won't help already closed issues/PRs though :/

Reply:
Split this out into 3:
- Closing PRs that merged as accepted
- Closing PRs that didn't merge as unresolved
- Closing issues as unresolved